internal/app/tfsec/custom/loader.go

```diff
@@ -0,0 +1,82 @@
+package custom
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"regexp"
+	"strings"
+)
+
+type ChecksFile struct {
+	Checks []*Check `json:"checks"`
+}
+
+func Load(rootPath string) error {
+	customCheckDir := fmt.Sprintf("%s/.tfsec", rootPath)
+	_, err := os.Stat(customCheckDir)
+	if os.IsNotExist(err) {
+		return nil
+	}
```

Comment:
```suggestion
	} else if err != nil {
	     return err
	}
```
in case the file isn't readable etc.